README.md

```diff
@@ -1 +1,3 @@
-# NewsAggregator
\ No newline at end of file
+# NewsAggregator
+
```

Comment:
A more detailed readme would be better.

- List the routes and maybe add curl requests to it